src/App.js

```diff
@@ -1,65 +1,39 @@
 import * as React from 'react';
+import TodoList from "./TodoList"
+import AddTodoForm from  "./AddTodoForm"
 
 
-
-const list = [
-  {
-    title: 'Read the book',
-    url:'https://courses.robinwieruch.de/p/the-road-to-learn-react',
-    link: '"The Road to React"',
-    id: 1,
-  },
-  {
-    title: 'Watch the tutorials',
-    url:'https://www.educative.io/',
-    link: '"React"',
-    id: 2, 
-  },
-  {
-    title: 'Complete assignment for this week',
-    url:'https://github.com/Code-the-Dream-School/ctd-react-sparrow/wiki',
-    link: '"Sparrow assignments"',
-    id: 3, 
-  },
-  {
-    title: 'Sign up for Mentor Sessions',
-    url:' https://docs.google.com/spreadsheets/d/1hbeVKsUxDUmPgD0cVZEL0xrkwdo-FBjtC1HEAPLz2bE/edit#gid=0//learn.codethedream.org/react-fundamentals-project-setup-and-react-basics/',
-    link: '"Sin Up Sheet"',
-    id: 4, 
-  },
-  {
-    title: 'Submit the assignment',
-    url:'https://github.com/dseydahmetova',
-    link: '"Github"',
-    id: 5, 
+function Search(){
+  const handleChange = (event) => {
+    console.log(event);
+  }
+  return(
+    <div>
+      <label htmlFor="search">Search: </label>
+      <input id="search" type = "text" onChange={handleChange} />
+    </div>
+  )
   }
 
-];
 
 
 function App() {
 
+ const [newTodo, setNewTodo] = React.useState('');
+
   return (
     <div>
     <h1>To do List for CTD</h1>
```

Comment:
nit: html tags should be indented.

Reply:
HI! I apologize for my emglish, what does mean indented? thanks

Reply:
Sorry, basically saying you could format your html. Like 
`<div>
    <h1> To do List for CTD </h1>
    <TodoList />
</div>`